ddp-workspace/projects/ddp-dsm-ui/src/app/sharedLearningUpload/services/sharedLearningsHTTP.service.ts

```diff
@@ -33,16 +34,13 @@ export class SharedLearningsHTTPService {
     return this.dsmService.getSomaticResultsFileUploadSignedUrl(this.selectedRealm, participantId, fileInformation);
   }
 
-  public upload(signedUrl: string, file: File): Observable<any> {
+  public upload(signedUrl: string, file: File, headers?: HttpHeaders): Observable<any> {
     this.dsmService.logToCloud(`Uploading file ${file.name} to url ${signedUrl}.`).subscribe({
-      next: data => {
```

Comment:
The earlier attempt to bypass the alert box didn't do the trick.  Now with the new header, we can skip the alert box when the logging fails.